### PR TITLE
Fixed Repeated word issue 

### LINKED ISSUE #5010: Repeated words in a line
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** At [https://publiclab.org/stats](https://publiclab.org/stats) there are two simultaneous ```and``` present.
There should be only one ```and```.

Selection_055

Update:

Replace https://github.com/publiclab/plots2/blob/master/config/locales/en.yml#L256 with
`<b>%{notes_count} notes</b> and <b>%{wikis_count} wiki edits</b>`

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by Saad2714**
Fixes #5010

Thanks!

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -253,7 +253,7 @@ en:
       notes_per_week: "%{note_count} notes per week over this period."
       edits_per_week: "%{count} edits per week over this period."
       notes_wikis_comments: |
-          <b>%{notes_count} notes</b> and <b>%{wikis_count} wiki edits and</b>
+          <b>%{notes_count} notes</b> and <b>%{wikis_count} wiki edits</b>
```
Comment (gauravano) [inline config/locales/en.yml]:
```suggestion
          <b>%{notes_count} notes</b>, <b>%{wikis_count} wiki edits</b>
```
Hey @cesswairimu @publiclab/plots2-reviewers, how about changing "and" to ","?

Comment (gauravano) [reply config/locales/en.yml]: We use notes_wikis_comments here only so it shouldn't be an issue too

Comment (cesswairimu) [reply config/locales/en.yml]: Yeah sounds good @gauravano @Saad2714

**Comment by sashadev-sky** [comment]
@gauravano looks like I dont have merge privileges on plots2 so leaving that to you!

**Comment by gauravano** [comment]
Merged :tada:. Thanks, everyone!

**Pull request merged**